sdk/anomalydetector/azure-ai-anomalydetector/tests/recordings/test_anomaly_detector.pyTestAnomalyDetectortest_entire_detect.json

```diff
@@ -0,0 +1,571 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://yxy-ad-uscx.cognitiveservices.azure.com//anomalydetector/v1.1/timeseries/entire/detect",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Accept-Encoding": "gzip, deflate",
+        "Connection": "keep-alive",
+        "Content-Length": "2856",
+        "Content-Type": "application/json",
+        "Ocp-Apim-Subscription-Key": "e5ed66969e5344628968af36253713f4",
```

Comment:
Your secrets are exposed. Please override it in `conftest.py` like: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/loadtestservice/azure-developer-loadtesting/tests/conftest.py